src/main.cpp

```diff
@@ -15,6 +20,15 @@ int main(int argc, char *argv[])
     app.setOrganizationDomain("Remote Control");
 
     QQmlApplicationEngine engine;
+
+    if(qEnvironmentVariableIntValue("SCORE_QML_REMOTE_DEBUG") > 0){
```

Comment:
it's in general better style to avoid code duplication: 

    const bool debugEnabled = qEnvironmentVariableIntValue("SCORE_QML_REMOTE_DEBUG") > 0;
    engine.rootContext()->setContextProperty("g_debugMessagesEnabled", debugEnabled);